### PR TITLE
[feature] initial implementation

### PR DESCRIPTION
**Pull request by trombik**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,80 @@
...
+| `packer_binary_url` | URL of remote file | `{{ packer_binary_base_url }}/{{ packer_binary_filename }}` |
+| `packer_binary_dir` | path to directory to install `packer` | `{{ __packer_binary_dir }}` |
+| `packer_binary_name` | file name of symlink to `packer` binary | `{{ __packer_binary_name }}` |
+| `packer_binary_path` | path to symlink | `{{ packer_binary_dir }}/{{ packer_binary_name }}` |
+| `packer_binary_owner` | owner of files and directories | `{{ __packer_binary_owner }}` |
+| `packer_binary_group` | group of files and directories| `{{ __packer_binary_group }}` |
+| `packer_binary_mode` | mode of `packer` binary | `755` |
+| `packer_binary_cache_dir` | path to directory where archives are kept | `{{ __packer_binary_cache_dir }}` |
+| `packer_binary_sha256sum` | list of SHA 256 sum of remote files | `{"packer_1.0.4_freebsd_amd64.zip"=>"8c6aebcaf228d6883cd6afda4b88028cd8168fd04d92feb016b7b4a79571d0a2", "packer_1.0.4_linux_amd64.zip"=>"646da085cbcb8c666474d500a44d933df533cf4f1ff286193d67b51372c3c59e", "packer_1.0.4_openbsd_amd64.zip"=>"487ca4f29c327b389370768cb303ade4d791854918d52632be305ac9e614e64f"}` |
```
Comment (mitsururike) [inline README.md:32]: is there any reason to use a dictionary instead of defining the default `__packer_binary_sha256sum` in vars/"platform".yml?

Comment (trombik) [reply README.md:32]: i did not yet bother, though, if the dict has filenames and sums of other versions, using non-default version is a matter of setting the version.

Comment (mitsururike) [reply README.md:32]: do you mean that you are planning to add the other versions to the dict so that users can choose the version?

Comment (trombik) [reply README.md:32]: yes for sure. there have been bugs that hit me hard and forced to install older one.

Comment (trombik) [reply README.md:32]: ideally, it should simply use distribution's package.

**Review by mitsururike — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (trombik) [reply README.md:32]: and that is why the role is named `ansible-role-packer-binary`

**Pull request merged**